### PR TITLE
Insight v5.5.14 is now released.

### PR DESCRIPTION
**Pull request by mtbc**
Update the Insight download URI.

### CODE REVIEW

File: binder/postBuild [hunk 1]
```diff
@@ -31,7 +31,7 @@ chmod +x xstartup
 # Desktop applications
 # Fiji and OMERO insight
 
-OMERO_INSIGHT_VERSION=5.5.11
+OMERO_INSIGHT_VERSION=5.5.14
 OMERO_INSIGHT_URL_BASE=https://github.com/ome/omero-insight/releases/download/v$OMERO_INSIGHT_VERSION
```
Comment (pwalczysko) [inline binder/postBuild:35]: Linkchecker is unhappy. Indeed, when I constructed the url as indicated here, this is

```
https://github.com/ome/omero-insight/releases/download/v5.5.14
```
This url is giving 404

**Review by pwalczysko — CHANGES_REQUESTED** [review]
Replace the link with a valid one please

File: binder/postBuild (hunk 1 above)
Comment (mtbc) [reply binder/postBuild:35]: The URL to check is the one of https://github.com/ome/omero-guide-fiji/blob/master/binder/postBuild#L43 -
```
OMERO_INSIGHT_VERSION=5.5.14
OMERO_INSIGHT_URL_BASE=https://github.com/ome/omero-insight/releases/download/v$OMERO_INSIGHT_VERSION
wget -q $OMERO_INSIGHT_URL_BASE/omero_ij-$OMERO_INSIGHT_VERSION-all.jar
```
works just fine.

I don't see that the linkchecker reports are related to this PR. Reading the travis log, it successfully does,
```
wget -q https://github.com/ome/omero-insight/releases/download/v5.5.14/omero_ij-5.5.14-all.jar
```

**Comment by jburel** [comment]
I think linkcheck errors are due to url like ``https://idr.openmicroscopy.org/webclient/?show=project-51`` I have replaced  it by ``https://idr.openmicroscopy.org/search/?query=Name:idr0062`` in some repos

**Comment by pwalczysko** [comment]
@mtbc @jburel Thanks for the explanation.

**Comment by pwalczysko** [comment]
@mtbc opened https://github.com/mtbc/omero-guide-fiji/pull/2 against this branch with @jburel 's suggested fix. Maybe if you merge, it will fix travis ?

**Comment by mtbc** [comment]
Maybe not "fix" but it looks better, thank you @pwalczysko.

**Review by pwalczysko — APPROVED** [review]

**Comment by mtbc** [comment]
Rerun of Travis seems to have solved the issue.

**Pull request merged**